engine/src/main/java/org/terasology/input/events/InputEvent.java

```diff
@@ -50,6 +58,10 @@ public Vector3i getTargetBlockPosition() {
         return targetBlockPosition;
     }
 
+    /**
+     *
+     * @return The time since the event was fired (also the game update loop's delta time).
```

Comment:
It's the delta of the update loop processing it, which is equal to the delta since the event was fired since all events are processed every loop.